src/V3AstNodeOther.h

```diff
@@ -2909,14 +2909,21 @@ class AstCoverDecl final : public AstNodeStmt {
     string m_linescov;
     int m_offset;  // Offset column numbers to uniq-ify IFs
     int m_binNum = 0;  // Set by V3EmitCSyms to tell final V3Emit what to increment
+    int m_begin;  // If the node corresponds to packed array, m_begin is `lo()`, m_end is
+                  // `hi() + 1`. In other cases they are both 0. It allows to distinguish `bit x`
+                  // from `bit [0:0] x`.
+    int m_end;
```

Comment:
Either 1: please use a VNumRange member instead, or ideally 2. have the code look at the dtypep()->skipRefp() of this node and based on the type operate appropriately.

Reply:
A dtype shouldn't change past V3TraceDecl where this would get set.